library/java/org/chromium/net/impl/NativeCronetEngineBuilderImpl.java

```diff
@@ -127,7 +127,7 @@ private EnvoyConfiguration createEnvoyConfiguration() {
         mDnsRefreshSeconds, mDnsFailureRefreshSecondsBase, mDnsFailureRefreshSecondsMax,
         mDnsQueryTimeoutSeconds, mDnsMinRefreshSeconds, mDnsPreresolveHostnames,
         mDnsFallbackNameservers, mEnableDnsFilterUnroutableFamilies, mEnableDrainPostDnsRefresh,
-        mEnableHttp3, mEnableHappyEyeballs, mEnableInterfaceBinding,
+        mEnableHttp3, true, mEnableHappyEyeballs, mEnableInterfaceBinding,
```

Comment:
nit: The pattern of passing all named values here is sort of convenient for remembering which parameter is which.